### PR TITLE
fix(docs site): remove 'insights' mentions

### PR DESCRIPTION
**Pull request by zuluecho9**
We noticed there were still quite a few 'Insights' references that shouldn't be there, so removed almost all bad ones (still some present I have questions out to teams about).

### CODE REVIEW

File: src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-260046.mdx [hunk 1]
```diff
@@ -16,7 +16,7 @@ For a list of known issues with the Python agent, see [Status of the Python agen
 
 ### New Feature
 
-* Custom Insights Events
+* Custom Events
```
Comment (paperclypse) [inline src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-260046.mdx:19]: Should we really be retroactively changing release notes?

File: src/content/docs/style-guide/writing-docs/article-templates/agent-api-guide-template.mdx [hunk 2]
```diff
@@ -326,7 +326,7 @@ APM includes a number of ways to record arbitrary custom data. For an explanatio
   <tbody>
     <tr>
       <td>
-        Send data about an event so you can analyze it in New Relic Insights...
+        Send data about an event so you can analyze it when querying...
```
Comment (paperclypse) [inline src/content/docs/style-guide/writing-docs/article-templates/agent-api-guide-template.mdx:329]: I'm intrigued and baffled by the ellipses in this doc, but it's outside the scope of this PR, I think

**Review by paperclypse — APPROVED** [review]
Excellent work! I left a couple of comments, one of which you can ignore :D

File: src/content/docs/style-guide/writing-docs/article-templates/agent-api-guide-template.mdx (hunk 2 above)
Comment (zuluecho9) [reply src/content/docs/style-guide/writing-docs/article-templates/agent-api-guide-template.mdx:329]: haha. yeah best left for another day

File: src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-260046.mdx (hunk 1 above)
Comment (zuluecho9) [reply src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-260046.mdx:19]: yeah probably not, you're right. i started changing them then stopped after a couple.

**Pull request merged**